lib/serializer.js

```diff
@@ -64,12 +64,9 @@ module.exports = class CustomSerializer extends Serializer {
      * @returns {Boolean}
      */
     _isSlotNode(node) {
-        if (node.name === 'slot') {
-            return true;
-        }
-        const attribs = node.attribs;
-        return node.name === 'script' &&
-            attribs && attribs.type === 'slot';
+        const { attribs, name} = node; 
```

Comment:
@vigneshshanmugam We can use `http://eslint.org/docs/rules/object-curly-spacing` for this but the problem is the existing code is not consistent for any of the `never` or `always` rules. It throws 58 and 60 errors respectively with `never` and `always` rules. Which one do you think should be used across? I am planning to add this and make it consistent